src/main/java/uk/gov/companieshouse/company_appointments/config/Config.java

```diff
@@ -28,14 +28,16 @@
 @Configuration
 public class Config implements WebMvcConfigurer {
     public static final String PATTERN_FULL_RECORD = "/**/full_record/**";
-    @Autowired
-    private RequestLoggingInterceptor loggingInterceptor;
-
-    @Autowired
-    private AuthenticationInterceptor authenticationInterceptor;
+    private final RequestLoggingInterceptor loggingInterceptor;
+    private final AuthenticationInterceptor authenticationInterceptor;
+    private final FullRecordAuthenticationInterceptor fullRecordAuthenticationInterceptor;
 
     @Autowired
-    private FullRecordAuthenticationInterceptor fullRecordAuthenticationInterceptor;
+    public Config (RequestLoggingInterceptor loggingInterceptor, AuthenticationInterceptor authenticationInterceptor, FullRecordAuthenticationInterceptor fullRecordAuthenticationInterceptor){
```

Comment:
Someone will have to explain this one to me - I see it all over the place. I thought the point of @Autowired on the private variables was that the constructor would then be generated automatically